docs/corruption-detection-and-recovery.md

```diff
@@ -0,0 +1,62 @@
+# etcd Corruption Detection and Recovery
+
+Currently, in etcd 3.5.0-3.5.2, there is a low probability risk of data corruption across the etcd members [Upstream Issue](https://github.com/etcd-io/etcd/issues/13766), [Openshift Documentation](https://access.redhat.com/solutions/6849521). There is a mitigation coming to the Openshift Cluster etcd Operator that enables the etcd 'experimental-initial-corrupt-check' which should help detect corruption and stop the bad member from joining the cluster.
+This document will outline some ways to verify if an etcd member is corrupted so that you can be more confident that the corruption recovery steps are necessary and should be taken.
+
+# Corruption Diagnosis
+Unfortunately, there's no single command that will give a precise 'corrupt/not corrupt' at this time; however, if most or all of the following return that there might be corruption, then it's likely that there is corruption and recovery steps should be taken. Please note, that a single failure in the following commands does not necessarily mean that corruption has occurred.
+
+## Check Prometheus for etcd revision divergence
```

Comment:
This is necessary, but not sufficient right?  In other words, if this reports ok, there is no need to progress further.

Reply:
Added a blurb calling this out